p2p/src/test/kotlin/maru/p2p/DiscoveryTest.kt

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright Consensys Software Inc.
+ *
+ * This file is dual-licensed under either the MIT license or Apache License 2.0.
+ * See the LICENSE-MIT and LICENSE-APACHE files in the repository root for details.
+ *
+ * SPDX-License-Identifier: MIT OR Apache-2.0
+ */
+package maru.p2p
+
+import java.util.concurrent.TimeUnit
+import maru.config.P2P
+import maru.consensus.ForkId
+import maru.p2p.discovery.MaruDiscoveryService
+import org.apache.tuweni.bytes.Bytes
+import org.assertj.core.api.Assertions.assertThat
+import org.awaitility.Awaitility
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.parallel.Execution
+import org.junit.jupiter.api.parallel.ExecutionMode
+
+@Execution(ExecutionMode.SAME_THREAD)
+class DiscoveryTest {
```

Comment:
Also, how this class is different from `MaruDiscoveryServiceTest`? It seems that they both test `MaruDiscoveryService`